src/pages/404.js

```diff
@@ -13,21 +16,18 @@
 
 
 import React from "react"
-import {Link} from "gatsby"
 import { AppAuthLayout } from "../features/app"
-//import HOME from "../features/navigation/navItems"
-import HOME from "../features/navigation"
-//https://www.lego.com/en-no/product/batman-classic-tv-series-batmobile-76188
+import {HOME} from "../features/navigation"
 
-const KrakenPage = () => {
+const NotFoundPage = () => {
     return (
       <>
         <AppAuthLayout title="Not Found">
 
-            <p>Jump in my old school batmobile and I'll give you a ride to my beauuuuutiful mansion for some good old fashioned fun.</p>
+            <p>Jump in my old school batmobile and I'll give you a ride to my beauuuuutiful mansion for some good old fashioned fun. Or....</p>
```

Comment:
This makes no sense without the image.

I want the first version of this to be boring, but working page. 
Let me know if I should do the copy @olavea.

You can go all out in #273, but it has to be on brand and I might never approve it.

Reply:
That's ok.
😺 
look at the [comment above](https://github.com/raae/pow-app/pull/269#discussion_r629935885)

I just needed to make a drawing after the coding, you know:

![Batman_on_the_beach](https://user-images.githubusercontent.com/7642493/118241931-04e65080-b49d-11eb-88bc-d7fff11759c3.jpeg)

[Photo](https://unsplash.com/photos/0xOCVe7nUU0) by [Zakaria Ahada](https://unsplash.com/@zakariahada) on [Unsplash](https://unsplash.com/@zakariahada).

An I have put some more links to batman LEGO photos from unsplash  in #273